### PR TITLE
Add the loading data with cleaning approach

### PR DESCRIPTION
**Pull request by peter279k**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

Adding the approach about loading data with cleaning mechanism and it's described in this [presentation](https://presentations.clickhouse.com/fosdem2023/).

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit a91da66961bfb90db7849c636c498067b482d9b7 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62690/a91da66961bfb90db7849c636c498067b482d9b7/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/getting-started/example-datasets/wikistat.md [hunk 1]
```diff
@@ -64,3 +64,22 @@ clickhouse-local --query "
...
+INSERT INTO wikistat WITH
+    parseDateTimeBestEffort(extract(_file, '^pageviews-([\\d\\-]+)\\.gz$')) AS time,
+    splitByChar(' ', line) AS values,
+    splitByChar('.', values[1]) AS projects
+SELECT
+    time,
+    projects[1] AS project,
+    projects[2] AS subproject,
+    decodeURLComponent(values[2]) AS path,
+    CAST(values[3], 'UInt64') AS hits
```
Comment (yariks5s) [inline docs/en/getting-started/example-datasets/wikistat.md:80]: the number of columns doesn't match: in the original table we have 6 rows, but we insert 5 rows🙂

**Review by yariks5s — CHANGES_REQUESTED** [review]

File: docs/en/getting-started/example-datasets/wikistat.md (hunk 1 above)
Comment (peter279k) [reply docs/en/getting-started/example-datasets/wikistat.md:80]: @yariks5s, thanks for your reply.

In the [presentation](https://presentations.clickhouse.com/fosdem2023/#17), the SQL about creating table is as follows:

```sql
CREATE TABLE wikistat
(
    time DateTime CODEC(Delta, ZSTD(3)),
    project LowCardinality(String),
    subproject LowCardinality(String),
    path String CODEC(ZSTD(3)),
    hits UInt64 CODEC(ZSTD(3))
)
ENGINE = MergeTree
ORDER BY (path, time);
```

That's different from original table schema and the `size` field is missed.

Should we be consistency for the table schema in the presentation?

Comment (yariks5s) [reply docs/en/getting-started/example-datasets/wikistat.md:80]: I think that we need to make it the same as in the presentation

**Review by yariks5s — APPROVED** [review]

**Pull request merged**